docs/content/guides/developer/nft.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Create a Non-Fungible Token
+keywords: [ERC-721, NFT]
+---
+
+On Sui, everything is an object. Moreover, everything is a non-fungible token (NFT) as its objects are unique, non-fungible, and owned. So technically, a basic type publishing is enough to create a specific NFT.
```

Comment:
`So technically, a basic type publishing is enough to create a specific NFT.` Had to reread this a few times. Can it be reworded or omitted?

Reply:
Good call